### ISSUE TITLE
[Screen Reader - Make Code - Help] Narrator focus does not synchronized with keyboard focus on all controls under 'Help' menu item.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2373]
User Experience:
Users who depend on Screen reader will not get the information about control if Narrator focus does not synchronize with control.

Repro-Steps:
1. Launch application URL : https://makecode.microbit.org/beta#
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Help" option appearing in header section and select it.
6. Navigate through all the controls of appearing after "Help".
7. Verify Narrator focus is not synchronized with keyboard focus or not under 'Help' menu item.

**Actual Result:**
Narrator focus is not synchronized with keyboard focus on all controls like (Support, Reference, Blocks, JavaScript etc) present under 'More' menu item on header section, due to this screen reader user is unable to interact with the control.

**Expected Result:**
Narrator focus should synchronized with keyboard focus on all controls like (Extensions, Print, Delete project, Report abuse) present under 'More' menu item on header section, so that screen reader user could easily interact with the control for further navigation.

**Note:**
This issue doesn't exist with Chrome + JAWS and NVDA.

**MAS Reference:**
MAS 4.2.10 - Tracking of Focus and Selection Attributes

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

[MAS4.2.10_Narrator focus is not sync.zip](https://github.com/microsoft/pxt-microbit/files/3480327/MAS4.2.10_Narrator.focus.is.not.sync.zip)

**Comment by jwunderl** [comment]
Same issue as https://github.com/microsoft/pxt-microbit/issues/2373, they're both the same dropdown element. Only seems to repro in edge

**Comment by sonimishra** [comment]
Issue has been verified and seems this issue is exact duplicate of the  #2362 
Hence closing the  bug.
